cluster/manifests/admission-control/teapot.yaml

```diff
@@ -15,6 +15,16 @@ webhooks:
         apiGroups: [""]
         apiVersions: ["v1"]
         resources: ["pods"]
+  - name: pod-binding-admitter.teapot.zalan.do
```

Comment:
This should be present only when `teapot_admission_controller_inject_environment_variables` is enabled.